homeassistant/components/motionblinds_ble/__init__.py

```diff
@@ -86,13 +92,40 @@ def async_update_ble_device(
 
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = device
 
+    # Register OptionsFlow update listener
+    entry.async_on_unload(entry.add_update_listener(options_update_listener))
+
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
+    # Apply options
+    await hass.async_create_background_task(
+        name=device.ble_device.address, target=apply_options(hass, entry)
+    )
```

Comment:
We create a task if its going to be awaited?

Reply:
Hm not sure why I awaited it. I use a background task since apply_options calls set_permanent_connection, which connects to the motor